recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ source:
     # 02xx - GPU only patch specific to open-ce (maybe)
 {% if build_type == 'cuda' %}
 #    - 0201-add-additional-cuda-rpaths.patch
-    - 0202-Remove-extra-null-terminators.patch
+#    - 0202-Remove-extra-null-terminators.patch
```

Comment:
Can we remove this line and delete the patch.

Reply:
Its Done